db/migrate/20230301083135_add_user_ref_to_comments.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToComments < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :author, null: false, foreign_key: true
+  end
+end
```

Comment:
- [ ] Kindly choose one migration between this file and [this one](https://github.com/soesandarwin2201/blog_with_ruby_on_rails/pull/1/files#diff-d250d1a7d57724a869bc988eb877c98cb6a06aec802de1a5a0e874a16608313e) to avoid errors. Delete the irrelevant one.